Assets/VRTK/Scripts/Interactions/Highlighters/VRTK_MaterialColorSwapHighlighter.cs

```diff
@@ -29,8 +29,8 @@ public class VRTK_MaterialColorSwapHighlighter : VRTK_BaseHighlighter
 
         private Dictionary<string, Material[]> originalSharedRendererMaterials = new Dictionary<string, Material[]>();
         private Dictionary<string, Material[]> originalRendererMaterials = new Dictionary<string, Material[]>();
-        private Dictionary<string, Coroutine> faderRoutines;
-        private bool resetMainTexture = false;
+        protected Dictionary<string, Coroutine> faderRoutines;
```

Comment:
Is there any reason these are being changed to protected?

I'm guessing this was an old change made in the file but it doesn't look required anymore?

Same goes for the other private to protected in this file.

Maybe worth just reverting the changes in this file?